corehq/warehouse/models/staging.py

```diff
@@ -0,0 +1,64 @@
+from django.db import models, transaction
+
+from dimagi.utils.couch.database import iter_docs
+
+from corehq.apps.groups.dbaccessors import get_group_ids_by_last_modified
+from corehq.apps.groups.models import Group
+
+
+class StagingTable(models.Model):
+
+    staged_on = models.DateTimeField(auto_now_add=True)
```

Comment:
just curious where this value would end up? i think we'd want to include batch id on this table as well